packages/web-app-user-management/src/views/Users.vue

```diff
@@ -409,9 +411,27 @@ export default defineComponent({
     },
     async editUser(editUser) {
       try {
-        await this.graphClient.users.editUser(editUser.id, editUser)
+        const actualUser = this.users.find((user) => user.id === editUser.id)
 
-        if (editUser.drive) {
+        const graphEditUserRawObjectExtractor = (user) => {
+          const extractUser = { ...user }
```

Comment:
probably easier to use `lodash-es/omit` for that :-)

Reply:
<3